src/EnergyPlus/ChillerElectricASHRAE205.cc

```diff
@@ -0,0 +1,1665 @@
+        }
+    } break;
+    default: {
+        assert(false);
+    } break;
+    }
+// Revisit
+#if 0
+        //        // If there is a fault of Chiller SWT Sensor
+//        if (this->FaultyChillerSWTFlag && (!state.dataGlobal->WarmupFlag) && (!state.dataGlobal->DoingSizing) && (!state.dataGlobal->KickOffSimulation)) {
+//            int FaultIndex = this->FaultyChillerSWTIndex;
+//            Real64 EvapOutletTempSetPoint_ff = EvapOutletTempSetPoint;
+//
+//            // calculate the sensor offset using fault information
+//            this->FaultyChillerSWTOffset = state.dataFaultsMgr->FaultsChillerSWTSensor(FaultIndex).CalFaultOffsetAct(state);
+//            // update the EvapOutletTempSetPoint
+//            EvapOutletTempSetPoint =
+//                    max(this->TempLowLimitEvapOut,
+//                        min(state.dataLoopNodes->Node(this->EvapInletNodeNum).Temp, EvapOutletTempSetPoint_ff - this->FaultyChillerSWTOffset));
+//            this->FaultyChillerSWTOffset = EvapOutletTempSetPoint_ff - EvapOutletTempSetPoint;
+//        }
+#endif // 0
+       // When implemented, TODO: correct temperature if using heat recovery
+
+    this->EvapMassFlowRate = state.dataLoopNodes->Node(this->EvapInletNodeNum).MassFlowRate;
+    // If some other component set the flow to 0, no reason to continue with calculations.
+    if (this->EvapMassFlowRate == 0.0) {
+        MyLoad = 0.0;
+        return;
+    }
+
+    Real64 CpEvap = FluidProperties::GetSpecificHeatGlycol(state,
+                                                           state.dataPlnt->PlantLoop(this->CWPlantLoc.loopNum).FluidName,
+                                                           state.dataLoopNodes->Node(this->EvapInletNodeNum).Temp,
+                                                           state.dataPlnt->PlantLoop(this->CWPlantLoc.loopNum).FluidIndex,
+                                                           RoutineName);
+
+    // Calculate mass flow rate based on MyLoad (TODO: then adjust it after determining if chiller can meet the load)
+    this->findEvaporatorMassFlowRate(state, MyLoad, CpEvap);
+
+    // Available chiller capacity is capacity at the highest sequence number; i.e. max chiller capacity
+    const Real64 maximumChillerCap = this->Representation->performance.performance_map_cooling
+                                         .calculate_performance(this->EvapVolFlowRate,
+                                                                this->EvapOutletTemp + DataGlobalConstants::KelvinConv,
+                                                                this->CondVolFlowRate,
+                                                                this->CondInletTemp + DataGlobalConstants::KelvinConv,
+                                                                this->MaxSequenceNumber)
+                                         .net_evaporator_capacity;
+    if (maximumChillerCap <= 0) {
+        // TODO: Issue a warning
```

Comment:
Should you issue a warning?

Reply:
Oops, meant to remove that check. Fixed.